app/controllers/split_checkout_controller.rb

```diff
@@ -65,23 +65,13 @@ def render_error
       replace("#flashes", partial("shared/flashes", locals: { flashes: flash }))
   end
 
-  def render_voucher_section_or_redirect
-    respond_to do |format|
-      format.cable_ready { render_voucher_section }
-      format.html { redirect_to checkout_step_path(:payment) }
-    end
```

Comment:
This was done to address [mkllnk](https://github.com/mkllnk) comment here https://github.com/openfoodfoundation/openfoodnetwork/pull/10587#discussion_r1163439737